ui/partials/perf/graphsctrl.html

```diff
@@ -23,11 +23,20 @@
           </td>
         </tr>
       </table>
-      <button class="btn btn-primary" ng-click="addTestData()">
+      <button class="btn btn-primary" ng-click="addTestData()" ng-hide="seriesList.length == 0">
         <span class="glyphicon glyphicon-plus" aria-hidden="true"></span> Add more test data</button>
     </div>
-    <div id="data-display">
```

Comment:
Without the previously-present "data-display" div the flex flow doesn't work correctly, you can see this if you try adding a graph.